### PR TITLE
Show clients on coordinator as well

### PR DESCRIPTION
**Pull request by nikifkon**
example output:

```
postgres=> show clients;
  client_id   |   user   |  dbname  |       server_id       | router_address 
--------------+----------+----------+-----------------------+----------------
 0xc0003ffad0 | postgres | postgres | no backend connection | localhost:7001
 0xc0000f09c0 | postgres | postgres | localhost:5001        | localhost:7000
 0xc0000f09c0 | postgres | postgres | localhost:5002        | localhost:7000
 0xc0003fef70 | postgres | postgres | no backend connection | localhost:7000
(4 rows)
```

### CODE REVIEW

File: coordinator/provider/coordinator.go [hunk 1]
```diff
@@ -32,6 +35,139 @@ import (
 	spqrparser "github.com/pg-sharding/spqr/yacc/console"
 )
 
+type FakeClient struct {
```
Comment (reshke) [inline coordinator/provider/coordinator.go:38]: can we just move FakeClient from router/client to pkg/client and reuse it?

File: pkg/client/clientpool.go [hunk 2]
```diff
@@ -6,8 +6,23 @@ import (
 	spqrlog "github.com/pg-sharding/spqr/pkg/spqrlog"
 )
 
+type RouterClient interface {
```
Comment (reshke) [inline pkg/client/clientpool.go:9]: Can we choose a better name? there is already struct with this exact name

**Review by reshke — CHANGES_REQUESTED** [review]

File: coordinator/provider/coordinator.go (hunk 1 above)
Comment (nikifkon) [reply coordinator/provider/coordinator.go:38]: Nice thought. Done. I also rename it to MockClient

File: pkg/client/clientpool.go (hunk 2 above)
Comment (nikifkon) [reply pkg/client/clientpool.go:9]: I think `ClientInfo` will suit, because it correspond to the same structure as in protobuf

**Review by reshke — APPROVED** [review]

**Pull request merged**